subprojects/junit5/junit5.gradle

```diff
@@ -0,0 +1,14 @@
+description = "Mockito JUnit 5 support"
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    compile project.rootProject
+    implementation libraries.junit5
```

Comment:
Let's reduce/change this dependency to `org.junit.jupiter:junit-jupiter-api:5.0.1` since we should be fine with the api for implementing the extension.
Additionally let's add `testRuntime libraries.junit5`.

Reply:
Thanks for these hints! 
Fixed!

Reply:
hmm, the implementation of the extension should just depend on the api.
Therefore we could change it to 
```
implementation 'org.junit.jupiter:junit-jupiter-api:5.0.1'
testRuntime 'org.junit.vintage:junit-vintage-engine:5.0.1'
```
isn't it?